packages/wallet/src/redux/sagas/adjudicator-watcher.ts

```diff
@@ -36,9 +36,9 @@ export function* adjudicatorWatcher(provider) {
 }
 
 function* dispatchEventAction(event: AdjudicatorEvent) {
+  const { channelId } = event;
```

Comment:
May as well destructure `eventType` here too

Reply:
I tried but looks like `eventType` doesn't have a specific structure, it's just an array of values which is why elsewhere in the code I had to use indexing to access its elements.